db/change_cache_test.go

```diff
@@ -277,17 +277,11 @@ func TestChannelCacheBufferingWithUserDoc(t *testing.T) {
 		t.Skip("This test does not work with XATTRs due to calling WriteDirect().  Skipping.")
 	}
 
-	base.EnableLogKey("Cache")
-	base.EnableLogKey("Cache+")
-	base.EnableLogKey("Changes")
-	base.EnableLogKey("Changes+")
-	base.EnableLogKey("DCP")
+	base.ConsoleLogLevel().Set(base.LevelDebug)
+	base.ConsoleLogKey().Set(base.KeyCache | base.KeyChanges | base.KeyDCP)
 	defer func() {
-		base.DisableLogKey("Cache")
-		base.DisableLogKey("Cache+")
-		base.DisableLogKey("Changes")
-		base.DisableLogKey("Changes+")
-		base.DisableLogKey("DCP")
+		base.ConsoleLogLevel().Set(base.LevelInfo)
+		base.ConsoleLogKey().Set(base.KeyNone)
```

Comment:
We should move this to restTester Close, so that (a) it gets run for every test, and (b) we don't have to remember to defer it in every test.